Diagnostic/mdsd/mdsd/CfgCtxManagement.cc

```diff
@@ -145,7 +145,7 @@ void CfgCtxIdentityComponent::Enter(const xmlattr_t& properties)
 	_ctxidentity = dynamic_cast<CfgCtxIdentity*>(ParentContext);
 	if (!_ctxidentity) {
 		FATAL("Found <IdentityComponent> in <" + ParentContext->Name() + ">; that can't happen");
-		IsValid = false;	// Bummer; invalid
```

Comment:
Also advise to remove from https://msazure.visualstudio.com/One/_git/Compute-Runtime-Tux?path=/mdsd/CfgCtxManagement.cc&version=GBhestolz/monmon-servicerequest&line=162&lineEnd=181&lineStartColumn=13&lineEndColumn=38&lineStyle=plain&_a=contents